src/components/Dialog.tsx

```diff
@@ -70,10 +71,12 @@ export function Header({ title, children, ruled }: HeaderProps) {
     <>
       <Column>
         <HeaderRow iconSize={1.2}>
-          <Row justify="flex-start" gap={0.5}>
-            <IconButton color="primary" onClick={useContext(OnCloseContext)} icon={ChevronLeft} />
-            <Row gap={0.5}>{title && <ThemedText.Subhead1>{title}</ThemedText.Subhead1>}</Row>
-          </Row>
+          <TextButton color="primary" onClick={useContext(OnCloseContext)}>
```

Comment:
nit: (I know this isn't a part of this PR) - hooks shouldn't be used in the render function